lib/blinkist-airbrake-scrubber/deep_traversal.rb

```diff
@@ -0,0 +1,33 @@
+# DeepTraversal provides traverse possibility of Hashes
+# Can handle Hash objects with deep nesting, or other nested objects that can be dug deeper (e.g. Array)
+class DeepTraversal
```

Comment:
This needs to be inside `Blinkist::Airbrake::Scrubber` potentially inside `Blinkist::Airbrake::Scrubber::Utils`